src/main/java/com/example/bookstoreapp/model/Book.java

```diff
@@ -9,10 +9,14 @@
 import java.math.BigDecimal;
 import lombok.Getter;
 import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
 
 @Entity
 @Getter
 @Setter
+@SQLDelete(sql = "UPDATE books SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
```

Comment:
"Where" annotation is deprecated, let's use SQLRestriction instead

Reply:
Agree